src/com/company/medicine/Appointment.java

```diff
@@ -0,0 +1,88 @@
+package com.company.medicine;
+
+import java.time.*;
+import java.util.Comparator;
+import java.util.Objects;
+
+public class Appointment {
+    protected LocalDateTime date;
+    protected Status status;
+    protected String specialization;
+    protected MedicalProcedure medicalProcedure;
+
+    public Appointment(LocalDate date, Integer hour, Integer minute, Status status, String specialization, MedicalProcedure medicalProcedure) {
+        this.date = LocalDateTime.of(date, LocalTime.of(hour, minute));
+        this.status = status;
+        this.specialization = specialization;
+        this.medicalProcedure = medicalProcedure;
+    }
+
+    public Appointment(LocalDateTime date, Status status, String specialization, MedicalProcedure medicalProcedure) {
+        this.date = date;
+        this.status = status;
+        this.specialization = specialization;
+        this.medicalProcedure = medicalProcedure;
+    }
+
+    public Appointment(Integer day, Integer month, Integer year,Integer hour, Integer minute, Status status, String specialization, MedicalProcedure medicalProcedure) {
```

Comment:
poti in constructorul asta sa apelezi constructorul de mai sus, fiindca seteaza aceleasi campuri, nu e nevoie sa duplici logica